### PR TITLE
Create portfolio page - mobile first

### PR DESCRIPTION
**Pull request by bhushan354**
- Added proper index.html file.
- Added proper style.css file.
- Used proper linters and fixed linters error.

Thank you ,
Bhushan

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,8 +43,6 @@
         <a href="#"><img src="mail.png" alt="E-Mail" class="navbar1" /></a>
       </nav>
     </header>
-
-    <!-- intropage -->
     <section class="mainSentences">
       <h1 class="mainSentenceText">
```
Comment (shahadat3669) [inline index.html:47]: ✅ Good job on the implementation! However, I noticed a couple of areas that require adjustment to match the design more closely:

1️⃣ The height of the "mainSentenceText" element appears to be larger than what is specified in the design. To ensure consistency, please update your code to match the designated height in the design.

2️⃣ Additionally, the margin-top for this heading should be set to 120px, as indicated in the design. Currently, it is set to 52px. Please modify your code accordingly to align with the design specifications.

By making these adjustments, you will ensure that your project reflects the intended design and maintains visual consistency. Keep up the great work! 👍🎉

File: index.html [hunk 2]
```diff
@@ -98,11 +96,15 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
     </section>
```
Comment (shahadat3669) [inline index.html:96]: ✅ Good work on the implementation! I have a couple of suggestions to further align your code with the design:

1️⃣ For the "connectIcons" list that contains social icons, the left margin is currently set to 22px. However, according to the design, it should be 24px. Additionally, the gap between the icons should be 16px, but it is currently set to 24px. Please update your code to reflect these specifications and ensure visual consistency with the design.

2️⃣ The margin-top for the "littleParagraph" element is currently set to 14px. As per the design, it should be 12px. Please make the necessary adjustment in your code to match the design specifications.

By making these adjustments, your project will closely adhere to the design guidelines, providing a consistent and visually appealing user experience. Keep up the great work! 👏🎉

File: index.html
Comment (shahadat3669) [inline index.html:1]: Great UI! 👍 However, there is a small issue with the vertical scrolling on the page. As per the design, there should be no vertical scrollbar visible. To fix this, you need to ensure that the content on the page fits within the viewport without requiring scrolling. 

[Bhushan-s-Portfolio.webm](https://github.com/bhushan354/My-Portfolio/assets/55840999/125b08ed-ee5b-4655-9d47-43551c34d3f9)

Please review the content, layout, and sizing of the elements to ensure they are properly adjusted to eliminate the vertical scrolling. By addressing this issue, you will achieve a more polished and seamless user experience. Keep up the good work! 💯

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @bhushan354 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Good PR Title
✅ Good PR Description
✅ All Linters Pass 👍 

## Required Changes ♻️

- [ ] 👤 Great job on including the "Authors" section in your project! However, I noticed that you have used the placeholder text "Author1" instead of your actual full name. To provide accurate and meaningful information to your users, I recommend updating the code to replace "Author1" with your full name. 🙏 
- [ ] I also noticed that you have repeated the same "Built With" section three times in your project. This repetition can be avoided for a cleaner and more concise README. Please keep only one instance of the "Built With" section and remove the duplicates.
- [ ] Additionally, since there is no FAQ section in your project, it is unnecessary to include the "FAQ" section title in the "Table of Contents" of your README file. This can potentially confuse visitors. Therefore, I recommend removing the "FAQ" section title from the "Table of Contents" to provide a more streamlined user experience.

Please make these necessary changes to your code, including updating the "Authors" section with your full name. Keep up the good work! 👏🌟

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I noticed that some of the project requirements, such as the README file, have already been implemented in the main branch. In future projects, it would be beneficial to commit the requirements in the feature branch rather than the main branch. This will make it easier for reviewers to review the changes and provide feedback specifically on the new features being added. Keeping the feature branch separate from the main branch helps in maintaining a clear and organized development process. Please keep this in mind for your next project.

Cheers and Happy coding!👏👏👏

## Have any doubt ❓


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## ⚠️ WARNING ⚠️
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bhushan354** [comment]
Hello sir ,
Thank you for reviewing my project, I have made necessary changes the previous reviewer suggested me to do.

Screenshot (75)

Also I fixed horizontal scrolling issue .

Thank you ,
Bhushan .

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @bhushan354          👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with the setup✔️       
  
# Required Changes  🌵  

- [x] Kindly remove the duplicate author 1 section as highlighted by the reviewer. Replace it with our name and this project will be done with :

### View:
image

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Cabraham1 — APPROVED** [review]
Hi @bhushan354 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**